### ISSUE TITLE
Multi camera calibration

### ISSUE THREAD

**Issue by Mymi-INS** [lambdaloop/anipose#95]
Hello,

We are using a large experimental setup to assess the memory performance in rats. Because of its large dimensions (4 x 2.5 m) and the low ceiling of our room, we need several cameras to cover the whole area (see the picture attached). We only need a 2D reconstruction of the animal’s position in the maze, but our issue is that the subject is only seen by 1 or 2 cameras at the same time, not by all of them.
Is there a way on Anipose 3D to calibrate all the cameras when there is not one common point seen by all of them?
Any other suggestion is also welcome :)

Thanks a lots!

![image](https://user-images.githubusercontent.com/61882756/183388908-091e082c-ee59-40aa-9403-6264724c7e9e.png)

**Comment by DestroytheCity** [comment]
Having a similar issue...have you found any ways to solve this?

**Comment by Mymi-INS** [comment]
> Having a similar issue...have you found any ways to solve this?

No, I still didn't manage to do it ! I will search and update you if I do !

**Comment by DestroytheCity** [comment]
So after speaking to the developers, it seems that for two cameras to understand their relationship to each other they need to see the same corners of the same CharUco board simultaneously. Our set up involves two sets of four cameras that are facing opposing directions, and so we are thinking of purchasing a larger board that would be seen by both sides at the same time.

**Comment by Mymi-INS** [comment]
Yeah we guessed that this will be the point. Do you know if we can calibrate 2 cameras by 2 cameras. Cause if all cameras has to see the same CharUco at the same time it will not work for me cause all my cameras are not seeing the same thing but I can arrange them to have them seeing the same thing 2 by 2.

**Comment by MahejabeenNidhi** [comment]
> Yeah we guessed that this will be the point. Do you know if we can calibrate 2 cameras by 2 cameras. Cause if all cameras has to see the same CharUco at the same time it will not work for me cause all my cameras are not seeing the same thing but I can arrange them to have them seeing the same thing 2 by 2.

Were you able to run it by calibrating the cameras 2 by 2?

**Comment by Mymi-INS** [comment]
No, I changed my cameras set up and now I have a software that record my two cameras simultaneously
